<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Widgets_ProbeGraphControl</class>
 <widget class="QWidget" name="Widgets_ProbeGraphControl" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="probesGroup" >
     <property name="title" >
      <string>Probes</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="ValidStateMarkingLineEdit" name="probeFilter" />
      </item>
      <item>
       <widget class="QListView" name="probes" >
        <property name="contextMenuPolicy" >
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="probeFunctions" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionDisplayProbeInfo" >
   <property name="text" >
    <string>Probe Info...</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>ValidStateMarkingLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
